packages/pages/src/dev/server/middleware/indexPage.ts

```diff
@@ -179,6 +181,16 @@ const createStaticPageListItems = (localDataManifest: LocalDataManifest) => {
   );
 };
 
+let accountId: string;
+let universe: string;
+
+if (fs.existsSync(".yextrc")) {
```

Comment:
Once Brian's [change](https://github.com/yext/pages/pull/357/files#diff-cedb5b2b607eb742283e66402bca0a5adf4a0f5b3a5839b123cde4c746ff5f49) has been merged can you change this to use the shared yextrc-reading function instead?